src/templates/Asciidoc/components/Content.js

```diff
@@ -378,6 +378,39 @@ const Content = styled.div`
     }
   }
 
+  .faq details {
+    summary {
+      font-weight: bold !important;
+      font-style: normal !important;
+      font-size: 120% !important;
+      padding-top: 15px;
+    }
+    .content {
+      padding: 0px 20px;
+    }
+  }
+
+  li details summary {
+    font-weight: normal !important;
+    font-style: italic;
+    font-size: 100% !important;
+  }
+
+  .singlespaced li p {
+    margin: 0 !important;
+    padding: 0 !important;
+  }
+
+  .callout1:before { content: "\\2460\\00a0"; font-size: 150%; }
```

Comment:
These annotation styles look really cool and I've been wanting something like them. They might need some CSS tweaks to align them with the text.

Reply:
Yay! We can figure out the tabbing later.